move-to-sdk/app/io/sphere/sdk/play/metrics/MetricAction.java

```diff
@@ -41,11 +38,31 @@ private void logRequestDataOnComplete(final Http.Context ctx, final List<ReportR
     }
 
     private Pair<List<ReportRawData>, List<ReportRawData>> splitByQueriesAndCommands(final List<ReportRawData> rawDatas) {
-        return ListUtils.partition(rawDatas, data -> data.getHttpRequest().getHttpMethod() == HttpMethod.GET);
+        return partition(rawDatas, data -> data.getHttpRequest().getHttpMethod() == HttpMethod.GET);
     }
 
     private Integer calculateTotalSize(final List<ReportRawData> rawDatas) {
         return rawDatas.stream().map(elem -> Optional.ofNullable(elem.getHttpResponse().getResponseBody())
                 .map(b -> b.length).orElse(0)).reduce(0, (a, b) -> a + b);
     }
+
+    /**
+     * Partitions <code>list</code> in two lists according to <code>predicate</code>.
+     * @param list the list which should be divided
+     * @param predicate returns true if the element of <code>list</code> should belong to the first result list
+     * @param <T> generic type of the list
+     * @return the first list satisfies <code>predicate</code>, the second one not.
+     */
+    public static <T> Pair<List<T>, List<T>> partition(final List<T> list, final Predicate<T> predicate) {
```

Comment:
I just copied this method from the one you removed from the JVM SDK because it was unused, which broke the Metrics: https://github.com/sphereio/sphere-jvm-sdk/commit/1f2a2efae3ff77ad66854ef5e4d7c595781aa283
I will create another ticket for you to improve it, but I don't want to pollute more this PR.